tasking/pulp/celery_app.py

```diff
@@ -11,6 +11,11 @@
 import time
 from gettext import gettext as _
 
+import os
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", 'pulp.app.settings')
+import django
```

Comment:
It would be good to add a whitespace between this import and the `os.environ.setdefault`